src/test/clojure/clara/test_fressian.clj

```diff
@@ -108,3 +108,14 @@
                      (serde (with-meta sm-custom {})))
             "cannot serialized custom sort comparators without name given in metadata")))))
 
+(deftest test-handler-identity
+  (let [v [1 2 3]
+        l (list 4 5 6)
+        m {:a 1 :b 2}
+        s #{:a :b :c}
```

Comment:
This seems to be missing some types, LazySeq for example.  I think it would be easy to add.
For example:
````
(let [ls (map inc [1 2]) ....
````

Reply:
added a few others as well